### ISSUE TITLE
removeMedia function can not operate

### ISSUE THREAD

**Issue by deepmk** [DevFactoryCH/media#3]
$this->public_path, $this->files_directory variable can not be found.

As a temporary solution 

private function removeMedia($media) {

 $this->public_path = rtrim(config('media.config.public_path'), '/\') . '/';
 $this->files_directory = rtrim(ltrim(config('media.config.files_directory'), '/\'), '/\') . '/';

```
  File::delete($this->public_path . $this->files_directory . $media->filename);
  $media->delete();
```

  }

**Comment by wunc** [comment]
+1
I just noticed this, too. (on 2.0 branch)

**Comment by markcameron** [comment]
Sorry didn't see this issue come through while I was on Holiday. Let me have a look.

**Comment by markcameron** [comment]
I think it's the bug I fixed in branch 1.0 here 4d8488c9cb550c70bc575f031dd8f954f48d8554

Cherry picked it and tagged a `2.0.3` release. Let me know if you have any more problems!

Thanks for the issue!
